### PR TITLE
feat(nexmark): add self-made nexmark queries in e2e test

### PR DESCRIPTION
**Pull request by lmatz**
## What's changed and what's your intention?
After #6758, we also add these queries into e2e test.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: e2e_test/streaming/nexmark/q101.slt.part [hunk 1]
```diff
@@ -0,0 +1,43 @@
+query ITI rowsort
+select * from nexmark_q101;
+----
+1000 mksxmeugsicmeb 73134520
```
Comment (BugenZhao) [inline e2e_test/streaming/nexmark/q101.slt.part:4]: Just to confirm, are these results generated by Postgres instead of RisingWave?

Comment (lmatz) [reply e2e_test/streaming/nexmark/q101.slt.part:4]: No, it's actually RW Batch's result.

As `auction_id` is created as the primary key and is always used as ether join key or group key,
I just compared the total number of rows manually against PG's result.

**Review by BugenZhao — APPROVED** [review]
LGTM

File: e2e_test/streaming/nexmark/q101.slt.part (hunk 1 above)
Comment (BugenZhao) [reply e2e_test/streaming/nexmark/q101.slt.part:4]: Okay. Let's try to be confident. 😄

**Review by fuyufjh — APPROVED** [review]

**Pull request merged**